components/Calender/.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
```

Comment:
it's uselss to have gitignore in a subfolder

Reply:
yes unless a subtree or module which it is not

Reply:
Its related to https://github.com/scbd/cbd-events/pull/1#discussion_r229599431.  How shall I resolve?